pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -238,8 +255,24 @@ func (c *Controller) startController(tc *corev1a1.FederatedTypeConfig) error {
 	return nil
 }
 
+func (c *Controller) startStatusController(tc *corev1a1.FederatedTypeConfig) error {
+	kind := tc.Spec.Template.Kind
+	statusKey := tc.Name + "/status"
```

Comment:
I'd like to see key creation factored into a helper and called here and in `reconcile` to simplify future maintenance.  Either that, or pass the key in from `reconcile`.